src/app/static/src/js/components/runWorkflow/runWorkflowCreate.vue

```diff
@@ -103,27 +103,17 @@
             },
             clone: function () {
                 if (this.selectedWorkflow && this.runWorkflowMetadata) {
-                    const {reports, git_branch, git_commit} = this.runWorkflowMetadata
-                    const clonedWorkflow: RunWorkflowMetadata = {
-                        ...emptyWorkflowMetadata,
-                        reports,
-                        git_branch,
-                        git_commit
+                    const clonedWorkflow = {
+                        ...workflowRunDetailsToMetadata(this.runWorkflowMetadata),
+                        name: "",
+                        instances: {}
```

Comment:
Why should instances used from the result of `workflowRunDetailsToMetadata`? That method copies them from the original metadata. I know the previous version of the code was not including instances, but I think it should have been, shouldn't it?

Reply:
I am not sure but according to the brief https://docs.google.com/document/d/1OicLFOm-Qcoos07iqLrXylPS2XBZ19EQDtlABjTFGJQ/edit#, instances should not be included.

Reply:
Yeah, I'm not sure it's preferable, but I think it's as we spec'ed it.